autogluon/training/docker/0.4.0/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,131 @@
+ARG CUDA_VER=11.2
+ARG LINUX_VER=ubuntu18.04
+ARG PYTHON_VERSION=3.8.12
+ARG PYTHON_VER=3.8
+ARG FROM_IMAGE=rapidsai/rapidsai-core
+
+FROM ${FROM_IMAGE}:cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER}
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG CUDA_VER
```

Comment:
Why is this needed when `ARG CUDA_VER=11.2` is defined at the top of the dockerfile?